Kitodo/src/test/java/org/kitodo/selenium/AddingST.java

```diff
@@ -158,6 +158,7 @@ public void addProcessesTest() throws Exception {
         assertEquals("Created Process has a wrong parent!", generatedProcess, generatedChildProcess.getParent());
     }
 
+    @Ignore
```

Comment:
I believe writing tests and ignoring them is not something that should be included in the general repository and should be used locally whenever possible.

Maybe there are other options for that, such as the use of test suites. In my opinion, this allows each test to be ignored if it's not working. Additionally, one might forget to re-add the test, and ultimately, it becomes dead code that should only be included in the application under specific circumstances.

At the very least, a comment explaining why and under what circumstances it is ignored should be included.